gossip/client_test.go

```diff
@@ -89,10 +89,10 @@ func TestClientGossip(t *testing.T) {
 		}
 	}()
 
-	if err := local.AddInfo("local-key", "local value", time.Second); err != nil {
+	if err := local.AddInfo("local-key", nil, time.Second); err != nil {
```

Comment:
Because the test passes without it; it's basically just a distraction to
leave it.
On Aug 26, 2015 5:16 PM, "Bram Gruneir" <notifications@github.com> wrote:

> In gossip/client_test.go
> <https://github.com/cockroachdb/cockroach/pull/2276#discussion_r38035903>:
>
> > @@ -89,10 +89,10 @@ func TestClientGossip(t *testing.T) {
> >  		}
> >  	}()
> >
> > -	if err := local.AddInfo("local-key", "local value", time.Second); err != nil {
> > +	if err := local.AddInfo("local-key", nil, time.Second); err != nil {
>
> why remove the content?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2276/files#r38035903>.
>